### DISCUSSION TITLE
Push Skript for Github Actions not working for Windows

### DISCUSSION THREAD

**Discussion by Bazzako** [cotes2020/jekyll-theme-chirpy#1343] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

Hi
Im having issues to get the website running on github pages. i think the only problem is the "pages-deploy.yml" since this is created for ubuntu. do you have any advice or solutions for windows users ?
locally its running just fine, just cant get it run on github pages with the github action

thanks you very much !

**Comment by NichtsHsu** [comment]
Maybe remove the `Gemfile.lock` from your github repo can solve your issue.
